app/src/components/sidebar/sidebar_group.module.sass

```diff
@@ -18,4 +20,18 @@
   transition: opacity 300ms
   &:hover,
   &:focus
-    opacity: .6
\ No newline at end of file
+    opacity: .6
+
+@include tabletMobile()
+  .root
+    &:last-child
+      margin-bottom: 57px
```

Comment:
![image](https://user-images.githubusercontent.com/47363177/112656049-ab2cb700-8e59-11eb-8a48-3133f2f87eda.png)

This not resolve the disappearance of the bottom padding when scrolling on firefox